<ui version="4.0" >
 <class>GlyphCustomClass</class>
 <widget class="QWidget" name="GlyphCustomClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>459</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>546</width>
    <height>459</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>546</width>
    <height>459</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Glyph Customization for Flat Displays</string>
  </property>
  <widget class="QGroupBox" name="groupBox_SortMode" >
   <property name="geometry" >
    <rect>
     <x>7</x>
     <y>10</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title" >
    <string>Sorting Mode</string>
   </property>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>17</x>
      <y>20</y>
      <width>491</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QRadioButton" name="radioButton_orig" >
       <property name="text" >
        <string>original</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_order" >
       <property name="text" >
        <string>ordered</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_datadriven" >
       <property name="text" >
        <string>data driven</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_derived" >
       <property name="text" >
        <string>derived</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_sortDim" >
   <property name="geometry" >
    <rect>
     <x>7</x>
     <y>70</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title" >
    <string>Sorting Dimension</string>
   </property>
   <widget class="QWidget" name="layoutWidget1" >
    <property name="geometry" >
     <rect>
      <x>17</x>
      <y>20</y>
      <width>491</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QCheckBox" name="checkBox_increase" >
       <property name="text" >
        <string>Increase</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_sortDim" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_datadriven" >
   <property name="geometry" >
    <rect>
     <x>7</x>
     <y>130</y>
     <width>531</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title" >
    <string>Data Driven Settings</string>
   </property>
   <widget class="QWidget" name="layoutWidget2" >
    <property name="geometry" >
     <rect>
      <x>7</x>
      <y>30</y>
      <width>501</width>
      <height>63</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QPushButton" name="pushButton_selectX" >
       <property name="text" >
        <string>select X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="lineEdit_dimX" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QPushButton" name="pushButton_selectY" >
       <property name="text" >
        <string>select Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="lineEdit_dimY" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_apply" >
   <property name="geometry" >
    <rect>
     <x>7</x>
     <y>420</y>
     <width>531</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_dims" >
   <property name="geometry" >
    <rect>
     <x>7</x>
     <y>250</y>
     <width>531</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
